pkg/registry/apis/folders/register.go

```diff
@@ -154,7 +155,9 @@ func (b *FolderAPIBuilder) PostProcessOpenAPI(oas *spec3.OpenAPI) (*spec3.OpenAP
 func (b *FolderAPIBuilder) GetAuthorizer() authorizer.Authorizer {
 	return authorizer.AuthorizerFunc(
 		func(ctx context.Context, attr authorizer.Attributes) (authorized authorizer.Decision, reason string, err error) {
-			if !attr.IsResourceRequest() || attr.GetName() == "" {
+			verb := attr.GetVerb()
+			name := attr.GetName()
+			if (!attr.IsResourceRequest()) || (name == "" && verb != "create") {
```

Comment:
I recently added constants for all verbs here https://github.com/grafana/grafana/blob/main/pkg/apimachinery/utils/verbs.go that we should prefer to use instead of hardcode them

Reply:
Nice! Switched them out.